astropy/utils/iers/iers.py

```diff
@@ -502,6 +502,7 @@ def read(cls, file=None, readme=None):
         # Read in as a regular Table, including possible masked columns.
         # Columns will be filled and converted to Quantity in cls.__init__.
         iers_a = Table.read(file, format='cds', readme=readme)
+        iers_a = Table(iers_a, masked=True, copy=False)
```

Comment:
This is a good example of what can break in application code and the suggested workaround.

Reply:
So just to make sure I fully understand, the value in setting ``masked=True`` is essentially auto-conversion from Column to MaskedColumn later, right?

Reply:
Yes.  The final version of `iers_a` is forced to have *all* `MaskedColumn`, and any new column will get added as `MaskedColumn`.